example/server/integrations.py

```diff
@@ -18,29 +22,83 @@ def poll_pending_deposits(cls, pending_deposits: QuerySet) -> List[Transaction]:
         Anchors should implement their banking rails here, as described
         in the :class:`.DepositIntegration` docstrings.
 
-        For the purposes of this reference implementation, we simply return
-        all pending deposits.
+        This implementation interfaces with a fake banking rails client
+        for demonstration purposes.
         """
+        # act like we're doing more work than we are for demo purposes
```

Comment:
Let's put this in the RailsClient, where the actual work would be done.  Perhaps say this is mimicking doing a fake API network request for your bank.

Reply:
actually i see why its out here, because we make N requests to `client.get_deposit`.  nabd.